wpeverest/everest-forms
# [Enhancement] - Add Search Fields on builder page

### All Submissions:






### Changes proposed in this Pull Request:

This PR aims to add support for the search field on the form builder page.

### How to test the changes in this Pull Request:

1. Checkout this PR branch.
2. Go to Add new form page.
3. You should see the search field just above fields.
4. Test the functionality, that's it!

### Types of changes:




### Other information:




### Changelog entry

> Enhancement -  Add search fields on builder page.
